pkg/virt-handler/vm_test.go

```diff
@@ -1833,6 +1835,68 @@ var _ = Describe("VirtualMachineInstance", func() {
 
 			controller.Execute()
 		}, 3)
+
+		It("code path check: startDomainNotifyPipe", func() {
+			domainPipeStopChan := make(chan struct{})
+			defer close(domainPipeStopChan)
+			vmi := api2.NewMinimalVMI("testvmi")
+			controller.startDomainNotifyPipe(domainPipeStopChan, vmi)
+		})
+
+		It("code path check: hasTargetDetectedDomain", func() {
+			vmi := api2.NewMinimalVMI("testvmi")
+			vmi.Status.MigrationState = &v1.VirtualMachineInstanceMigrationState{}
+			vmi.Status.MigrationState.EndTimestamp = &metav1.Time{
+				Time: time.Now(),
+			}
+			controller.hasTargetDetectedDomain(vmi)
+		})
+
+		It("code path check: updateGuestInfoFromDomain", func() {
+			domain := api.NewMinimalDomainWithUUID("testvmi", vmiTestUUID)
+			domain.Status.OSInfo = api.GuestOSInfo{
+				Name: "new-name",
+			}
+			vmi := api2.NewMinimalVMI("testvmi")
+			vmi.Status.GuestOSInfo = v1.VirtualMachineInstanceGuestOSInfo{
+				Name: "old-name",
+			}
+			controller.updateGuestInfoFromDomain(vmi, domain)
+		})
```

Comment:
These tests do not actually test anything. They don't expect any values nor validate order of execution or anything like that.
Am I missing anything, or is it just pure-raising-coverage code?

IMHO - the bare minimum here is to at least test the values that these functions return.

Reply:
(As a side note - see that all the error checking within these functions are not being covered - which is totally fine by me :) )

Reply:
Hey, thanks for the review!

Actually, these are code path tests for the "happy path", so I didn't even bother checking the return values. At least those make sure the code doesn't fail :) . But yes, I can add those verifications.